nautobot/circuits/filters.py

```diff
@@ -198,7 +177,6 @@ class Meta:
 class CircuitTerminationFilterSet(
     BaseFilterSet,
     CableTerminationModelFilterSetMixin,
-    LocatableModelFilterSetMixin,
```

Comment:
Why remove this and manually declare a `location` filter? Should be able to just update the mixin class instead.

Reply:
Yeah I was gonna do this one step at a time, If I remove the Site and Region filters from the mixin class, the tests are just gonna break all at once.

Reply:
Gotcha. Sounds good.